.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -45,16 +45,16 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: DetektReportsRxJava
-          path: RxJavaTemplate/build/reports/detekt/
+          path: RxJavaTemplate\[DEPRECATED\]/build/reports/detekt/
 
       - name: Run unit tests and Jacoco on RxJavaTemplate
-        working-directory: ./RxJavaTemplate
+        working-directory: "./RxJavaTemplate[DEPRECATED]"
         run: ./gradlew jacocoTestReport
 
       - name: Archive code coverage reports on RxJavaTemplate
         uses: actions/upload-artifact@v2
         with:
           name: CodeCoverageReportsRxJava
           path: |
-            RxJavaTemplate/app/build/reports/jacoco/jacocoTestReport/
-            RxJavaTemplate/data/build/reports/jacoco/jacocoTestReport/
+            RxJavaTemplate\[DEPRECATED\]/app/build/reports/jacoco/jacocoTestReport/
+            RxJavaTemplate\[DEPRECATED\]/data/build/reports/jacoco/jacocoTestReport/
```

Comment:
do you think we should allow the time on our CI for it to verify anymore, given it's deprecated? 💭

Reply:
Yeah, I agree. 

**Advantage**: Faster CI builds that are only focused on CoroutineTemplate
**Disadvantage**: Possible chance for RxJavaTemplate to break without us knowing

I'm all in favor of simply removing it from the CI completely 🙏